app/src/main/java/com/bootcamp_android/parking_app/Lot.kt

```diff
@@ -1,6 +1,6 @@
 package com.bootcamp_android.parking_app
 
- data class Lot (val id : String, val date: String , val hour:String){
```

Comment:
Create a folder to put your entities (Entities related to the business, like "Lot" for example)